.gitmodules

```diff
@@ -12,7 +12,7 @@
 	url = https://github.com/eigenteam/eigen-git-mirror.git
 [submodule "ext/spdlog"]
 	path = ext/spdlog
-	url = https://github.com/gabime/spdlog.git
+	url = https://github.com/pramodk/spdlog.git
 [submodule "ext/fmt"]
 	path = ext/fmt
-	url = https://github.com/fmtlib/fmt.git
+	url = https://github.com/pramodk/fmt.git
```

Comment:
Have to see this : PR upstream is pending. So most likely we have to fork under BlueBrain.